unittests/failuretests.py

```diff
@@ -242,12 +242,12 @@ def test_dependency_invalid_method(self):
         dep = declare_dependency(dependencies : zlib_dep)
         dep.get_pkgconfig_variable('foo')
         '''
-        self.assertMesonRaises(code, "Method.*pkgconfig.*is invalid.*internal")
+        self.assertMesonRaises(code, ".*is not a pkgconfig dependency")
```

Comment:
Should there be a space between `*` and `is`? I guess this doesn't matter too much, as this is a regex, but its different from line 250...

Reply:
There was no space in the previous regex, I don't think it matters tbh.